src/Actions/Family/ListFamilies.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Choredo\Actions\Family;
+
+use Assert\Assert;
+use Choredo\Entities\Family;
+use Choredo\EntityManagerAwareInterface;
+use Choredo\Filter;
+use Choredo\Filterable;
+use Choredo\HasEntityManager;
+use Choredo\Output\CreatesFractalScope;
+use Choredo\Output\FractalAwareInterface;
+use Choredo\Pageable;
+use Choredo\Pagination;
+use Choredo\PaginationCriteria;
+use Choredo\Repositories\FamilyRepository;
+use Choredo\Sortable;
+use Choredo\Transformers\FamilyTransformer;
+use Doctrine\ORM\Tools\Pagination\Paginator;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Zend\Diactoros\Response\JsonResponse;
+use const Choredo\DAYS_OF_WEEK;
+use const Choredo\REQUEST_FILTER;
+use const Choredo\REQUEST_PAGINATION;
+use const Choredo\REQUEST_SORT;
+
+class ListFamilies implements FractalAwareInterface, EntityManagerAwareInterface, Pageable, Sortable, Filterable
+{
+    use CreatesFractalScope;
+    use HasEntityManager;
+
+    public static function getSortableFields(): array
+    {
+        return [
+            'createdDate',
+        ];
+    }
+
+    public static function getDefaultLimit(): int
+    {
+        return Pageable::DEFAULT_LIMIT;
+    }
+
+    public static function getMaxLimit(): int
+    {
+        return Pageable::DEFAULT_MAX_LIMIT;
+    }
+
+    public static function getDefaultSort(): array
+    {
+        return [
+            ['createdDate', 'ASC'],
```

Comment:
I think there might be an ASC constant. http://php.net/manual/en/array.constants.php